src/Tabs.php

```diff
@@ -282,23 +332,63 @@ private function renderPanes(array $panes): string
             : '';
     }
 
+    /**
+     * Prepare Nav::widget for using
+     *
+     * @param array $options
+     * @param array $items
+     *
+     * @return Nav
+     */
+    private function prepareNav(array $options, array $items): Nav
+    {
+        $definitions = [];
+
+        foreach ($options as $name => $value) {
+            $definitions[$name . '()'] = [$value];
+        }
+
+        return Nav::widget($definitions)->items($items);
```

Comment:
I think we should add another `navConfig()` method, to separate the widget's configuration methods, from the widget's attributes.